src/Apps/Conversation/Routes/Conversation/index.tsx

```diff
@@ -3,23 +3,32 @@ import { Conversation_me } from "__generated__/Conversation_me.graphql"
 import { AppContainer } from "Apps/Components/AppContainer"
 import { ConversationFragmentContainer as Conversation } from "Apps/Conversation/Components/Conversation"
 import { SystemContext } from "Artsy"
+import { findCurrentRoute } from "Artsy/Router/Utils/findCurrentRoute"
 import { ErrorPage } from "Components/ErrorPage"
+import { Match } from "found"
 import React, { useContext } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { userHasLabFeature } from "Utils/user"
 
 interface ConversationRouteProps {
   me: Conversation_me
   conversationID: string
+  match: Match
 }
 
 export const ConversationRoute: React.FC<ConversationRouteProps> = props => {
   const { me } = props
   const { user } = useContext(SystemContext)
   const isEnabled = userHasLabFeature(user, "User Conversations View")
   if (isEnabled) {
+    const route = findCurrentRoute(props.match)
+    let maxWidth
+
+    if (route.displayFullPage) {
```

Comment:
I'm gonna timebox some time soon to add proper types to our routes

Reply:
That would be super sweet. Always happy to pair.